vmm/src/config.rs

```diff
@@ -441,6 +450,65 @@ impl<'a> DeviceConfig<'a> {
     }
 }
 
+pub struct VhostUserNetConfig<'a> {
+    pub mac: MacAddr,
+    pub sock: &'a Path,
+    pub num_queue_pairs: usize,
```

Comment:
Sounds good! I'd like to put the new struct into vm-virtio/src/vhost_user/mod, as one public struct of vhost_user, then, vmm/src/config could refer to it for use. This change will decrease the parameter number for a series of functions in config.rs, vm.rs and device implementation file. But I think it's no need to abstract the value parse process for those fields because of the type change and other fields involved.